pytorch_lightning/loggers/wandb.py

```diff
@@ -122,16 +119,18 @@ def __init__(
             )
 
         super().__init__()
-        self._name = name
-        self._save_dir = save_dir
         self._offline = offline
-        self._id = version or id
-        self._anonymous = 'allow' if anonymous else None
-        self._project = project
         self._log_model = log_model
         self._prefix = prefix
         self._experiment = experiment
-        self._kwargs = kwargs
+        # set wandb init arguments
+        self._wandb_init = dict(resume='allow', id=version or id, dir=save_dir,
+                                anonymous='allow' if anonymous is True else None if anonymous is False else anonymous)
```

Comment:
can we rather define this as look-up-table aka dictionary?
```
an_lut = {True: 'allow', False: None}
anonymous = an_lut.get(anonymous, anonymous)
```

Reply:
Great idea!